### PR TITLE
pfring: fix compile warning

### PR DESCRIPTION
**Pull request by victorjulien**

### CODE REVIEW

File: src/runmode-pfring.c [hunk 1]
```diff
@@ -264,10 +264,13 @@ static void *ParsePfringConfig(const char *iface)
                 }
             }
         } else {
-            if (StringParseUint16(&pfconf->threads, 10, 0, (const char *)threadsstr) < 0) {
+            uint16_t threads = 0;
```
Comment (catenacyber) [inline src/runmode-pfring.c:267]: Out of curiosity, why not change the definition of `threads` in `PfringIfaceConfig` structure ?

Comment (victorjulien) [reply src/runmode-pfring.c:267]: Would be good to do. This is just quick to get rid of the warning.

**Comment by victorjulien** [comment]
Replaced by #5234

File: src/runmode-pfring.c (hunk 1 above)
Comment (catenacyber) [reply src/runmode-pfring.c:267]: Ok thanks

**Pull request closed without merging**